nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/StandardNiFiServiceFacade.java

```diff
@@ -1462,6 +1459,25 @@ private Set<AffectedComponentEntity> getComponentsAffectedByParameterContextUpda
         return dtoFactory.createAffectedComponentEntities(affectedComponents, revisionManager);
     }
 
+    private void getAffectedComponentsByParameterUpdate(final ParameterContextDTO parameterContextDto, final Set<String> updatedParameterNames, final Set<ComponentNode> affectedComponents,
```

Comment:
Perhaps this should be `getComponentsAffectedBy...`

Reply:
This method has been removed.